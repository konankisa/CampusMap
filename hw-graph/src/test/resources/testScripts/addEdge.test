# Creates an empty graph, adds 2 node to it,
# and adds an edge between the 2 nodes

# Create a graph
CreateGraph graph

# Add a node
AddNode graph node1

# Add another node
AddNode graph node2

# Add an edge from node1 to node2 in graph
AddEdge graph node1 node2 edge