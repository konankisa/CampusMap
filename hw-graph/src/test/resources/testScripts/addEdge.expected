# Creates an empty graph, adds 2 node to it,
# and adds an edge between the 2 nodes

# Create a graph
created graph graph

# Add a node
added node node1 to graph

# Add another node
added node node2 to graph

# Add an edge from node1 to node2 in graph
added edge edge from node1 to node2 in graph